### ISSUE TITLE
Release 1.1.1 to fix the jupyterhub Docker images

### ISSUE THREAD

**Issue by consideRatio** [jupyterhub/jupyterhub#2911]
I think solving https://github.com/jupyterhub/jupyterhub/issues/2852 that broke the jupyterhub/jupyterhub:1.1.0 image merits a 1.1.1 release. We have the jupyterhub/jupyterhub:latest working again since #2905.

We should perhaps first ensure the build docker image works with tests in the future. I don't yet overview all tests we run though, but here is a test I used to reproduce the #2852 issue.

```
# using a pre-build image
IMG=jupyterhub/jupyterhub:latest
docker run -dt --rm -p 8000:8000 --name jh $IMG
sleep 3
curl localhost:8000/hub/login
docker logs jh
```

```
# building an image also
cd /tmp
git clone https://github.com/jupyterhub/jupyterhub.git
cd jupyterhub
IMG=jh
docker build -t $IMG .
docker stop jh || true
docker run -dt --rm -p 8000:8000 --name jh $IMG
sleep 3
curl localhost:8000/hub/login
docker logs jh
```

**Comment by manics** [comment]
We could add a docker build stage to the travis job, but as you say we'd still need to figure out how to test it.

**Comment by willingc** [comment]
Ping @minrk. What is needed to get this out?

**Comment by analytically** [comment]
1.1 is broken and unusable for container setup, a fix should be released asap.

**Comment by manniche** [comment]
Using [jpuyterhub:latest](https://hub.docker.com/layers/jupyterhub/jupyterhub/latest/images/sha256-f60ae006f652714dcee37ed1f18fade14a813a16717b2d8bd379def1556b698c?context=explore) today works

**Comment by BobCashStory** [comment]
Need an update too

**Comment by vlizanae** [comment]
jupyterhub:latest fetches 1.2, while 1.2 is not even on PyPI. Is there a reason there's no stable tag?

**Comment by manics** [comment]
@vlizanae that's a separate issue to this one, see https://github.com/jupyterhub/jupyterhub/issues/2848

**Comment by analytically** [comment]
So is there a stable image that works?

**Comment by vlizanae** [comment]
With Docker/Swarm spawner no, I don't think so.

**Comment by yuvipanda** [comment]
Is there an update on this?

**Comment by consideRatio** [comment]
I verified we have a published a image that works in at least 1.2.2 and 1.3.0. I could not reproduce the error in those images.
